TypeScriptReact.YAML-tmLanguage

```diff
@@ -28,9 +28,23 @@ repository:
     patterns:
     - include: '#jsx-tag-without-attributes-in-expression'
     - include: '#jsx-tag-in-expression'
-    - include: '#jsx-tag-invalid'
+    - include: '#jsx-fragment'
 
   #jsx tags
+  jsx-fragment:
+    name: meta.tag.fragment.tsx
```

Comment:
Instead of adding new rule for jsx-fragment, 
Update: jsx-tag-without-attributes-in-expression and jsx-tag-without-attributes as:
```yaml
  #jsx tags
  jsx-tag-without-attributes-in-expression:
    begin: |-
      (?x)
        (?<=[({\[,?=>:*]|&&|\|\||\?|\Wreturn|^return|\Wdefault|^)\s*
        (?=(<)\s*((?:[a-z][a-z0-9]*|([_$a-zA-Z][-$\w.]*))(?<!\.|-))?\s*(>))
    end: (?!\s*(<)\s*((?:[a-z][a-z0-9]*|([_$a-zA-Z][-$\w.]*))(?<!\.|-))?\s*(>))
    patterns:
    - include: '#jsx-tag-without-attributes'

  jsx-tag-without-attributes:
    name: meta.tag.without-attributes.tsx
    begin: (<)\s*((?:[a-z][a-z0-9]*|([_$a-zA-Z][-$\w.]*))(?<!\.|-))?\s*(>)
    end: (</)\s*((?:[a-z][a-z0-9]*|([_$a-zA-Z][-$\w.]*))(?<!\.|-))?\s*(>)
```

Reply:
Updated.